src/Algolia.Search.Test/EndToEnd/Index/SettingsTest.cs

```diff
@@ -131,7 +132,7 @@ public async Task TestSettings()
             saveSettingsResponse.Wait();
 
             var getSettingsResponse = await _index.GetSettingsAsync();
-            var spceficPropertiesCheck = new List<string> {"Version", "AlternativesAsExact", "DecompoundedAttributes"};
+            var spceficPropertiesCheck = new List<string> { "Version", "AlternativesAsExact", "DecompoundedAttributes" };
```

Comment:
nit: next time, let's include those formatting changes in a different commit ;)

Reply:
Totally agree.
I will add an editor.config. VSCode and resharper doesn't have the same formatting rule ><